services/galley/src/Galley/Cassandra/TeamFeatures.hs

```diff
@@ -96,14 +96,24 @@ setFeatureStatusNoConfig ::
   Proxy a ->
   TeamId ->
   TeamFeatureStatus 'WithoutLockStatus a ->
+  Maybe TeamFeatureTTLValue ->
   m (TeamFeatureStatus 'WithoutLockStatus a)
-setFeatureStatusNoConfig _ tid status = do
+setFeatureStatusNoConfig _ tid status ttl = do
   let flag = tfwoStatus status
   retry x5 $ write insert (params LocalQuorum (tid, flag))
   pure status
   where
+    minutes = (* 60)
```

Comment:
Do we already have a way to handle this kind of stuff in the codebase?

Reply:
I am not aware

Reply:
i found https://github.com/wireapp/wire-server/blob/4c2dc58c75ce25da228f6e9dc1069a82e628fc26/services/brig/src/Brig/Unique.hs#L68, but it's not used exactly exhaustively...